cypress/integration/metaSpec.js

```diff
@@ -0,0 +1,27 @@
+describe('The example page with imgix meta tags', () => {
+  it('successfully loads', () => {
+    cy.visit('/cypress/fixtures/samplePage.html');
+    // .should('equal', 'ix:useHttps');
+    cy.get('meta').each(($el, index, $list) => {
+      const property = $el.attr('property');
+      if (!!property) {
+        expect(property).to.include('ix');
+      }
+    });
+  });
+  it('successfully applies imgix paramenters to the source url', () => {
+    cy.get('img').each(($el, index, $list) => {
```

Comment:
Nit: we should remove `$list` if it's not getting referenced in the callback fn
```suggestion
    cy.get('img').each(($el, index) => {
```